.vscode/settings.json

```diff
@@ -4,11 +4,5 @@
     "**/src/**/*.js": "javascriptreact",
     "**/src/**/*.jsx": "javascriptreact",
     "**/.husky/": "bash"
-  },
```

Comment:
You can also just do away with these modifications on this file.

Reply:
Done